app/helpers/application_helper.rb

```diff
@@ -205,8 +205,8 @@ def searchable?
   end
 
   def auto_complete_search(name, val, options = {})
-    path = options[:path] || send("#{controller_name}_path")
```

Comment:
You shouldn't remove this condition.  While it might not get used on the filters page because :path is supplied, we use auto_complete_search throughout the rest of the app and it's in constant use.